docs/streams.md

```diff
@@ -148,8 +145,6 @@ consideration:
 
 - Due to the nature of `$all`, projections using `fromAll` read any deleted events that have not been
   scavenged. They also receive any tombstone events from hard deletes.
-- Projections that read from a specific stream receive that stream's metadata events. You can filter these out
```

Comment:
we're 100% sure is does not _still_ apply in 20.10 ?

Reply:
tbh im not sure it ever applied, unless you know otherwise?